server/pulp/plugins/loader/api.py

```diff
@@ -411,17 +411,36 @@ def get_cataloger_by_id(catloger_id):
 
 def load_content_types(types_dir=_TYPES_DIR, dry_run=False, drop_indices=False):
     """
-    :type types_dir: str
+    Check or update database with content unit types information.
+
+    :param types_dir: path to content unit type JSON files,
+                      currently used only for node.json
+    :type types_dir:  str
```

Comment:
Personal preference thing (do whatever you want, this is not a blocker for merging) but I prefer putting two spaces between type and the variable. This way the variable names line up and it is easier for me to parse visually.